package.json

```diff
@@ -2,7 +2,8 @@
   "name": "root",
   "private": true,
   "scripts": {
-    "install-all": "lerna exec npm install; npm install",
+    "prepare": "lerna run prepare",
+    "install-all": "npm install; lerna run prepare; lerna exec npm install",
```

Comment:
This is the main change - Lerna doesn't run prepare when installing (which is weird!)